articles.tex

```diff
@@ -1,212 +1,316 @@
-        \item Foster mutual assistance in the honorable labors and aspirations of life
-        \item Recognize devotion to unsullied friendship and unfaltering fidelity as objects worthy of the highest aim and purpose of associated effort
+	\item Promote the moral and social development of its members
+	\item Develop each individual into proactive leaders
+	\item Encourage the cultivation of intellect and ties to the fraternity's
+		values
+	\item Foster mutual assistance in the honorable labors and aspirations of
+		life
+	\item Recognize devotion to unsullied friendship and unfaltering fidelity as
+		objects worthy of the highest aim and purpose of associated effort
 \end{itemize}
 
 \section{Governing Guidelines}
+\label{sec:governing-guidelines}
+
 This chapter shall be governed:
 \begin{itemize}
-        \item By this edition of the chapter’s Constitution and Bylaws
-        \item By the Code of the General Fraternity of Beta Theta Pi
-        \item By the laws of the Interfraternity Council of the Worcester Polytechnic Institute
-        \item By the policies of Worcester Polytechnic Institute
-        \item By all federal, state, and local laws
+	\item By this edition of the chapter’s Constitution and Bylaws
+	\item By the Code of the General Fraternity of Beta Theta Pi
+	\item By the laws of the Interfraternity Council of the Worcester
+		Polytechnic Institute
+	\item By the policies of Worcester Polytechnic Institute
+	\item By all federal, state, and local laws
 \end{itemize}
 
-\article{Membership Qualifications and Expectations}
+\chapter{Membership Qualifications and Expectations}
+\label{cha:membership-qualifications-and-expectations}
 
 \section{Membership Qualifications}
+\label{sec:membership-qualifications}
 
-Membership shall be conferred only upon male students of Worcester Polytechnic Institute who have met the qualifications set out in Article II of the Code of Beta Theta Pi ordained “Membership”.
-The aforementioned must also fulfill such additional scholastic requirements as may be prescribed by this Constitution or by this chapter’s Bylaws.
-
+Membership shall be conferred only upon male students of Worcester Polytechnic
+Institute who have met the qualifications set out in Article II of the Code of
+Beta Theta Pi ordained ``Membership''.
+The aforementioned must also fulfill such additional scholastic requirements as
+may be prescribed by this Constitution or by this chapter’s Bylaws.
 
 \subsection{Zero Tolerance Policy for Harassment and Discrimination} 
+\label{sec:zero-tolerance-policy-for-harassment-and-discrimination}
 
-The Beta Theta Pi chapter will not discriminate against individuals based upon race, age, citizenship, origin, ancestry, religion, disabilities, sexual orientation, or marital status.
+The Beta Theta Pi chapter will not discriminate against individuals based upon
+race, age, citizenship, origin, ancestry, religion, disabilities, sexual
+orientation, or marital status.
```

Comment:
Not an issue with your work, but this subsection really doesn't fit in where it is. Oh well

Reply:
Yeah the constitution still has a bunch of issues. We can't really change anything here though without an amendment